### DISCUSSION TITLE
Welcome to aws-java-nio-spi-for-s3 Discussions!

### DISCUSSION THREAD

**Discussion by markjschreiber** [awslabs/aws-java-nio-spi-for-s3#180] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by stefanofornari** [comment]
Hello, I am Stefano and I wanted to use aws-java-nio in my own projects to access S3 services, not necessarily on AWS. I ended up contributing this functionality and I hope to keep helping the evolution of this great project!

**Comment by thsandu** [comment]
Hi, I am Thomas and we started doing more extensive integration tests in our projects. Tests are covering multiple file systems. To be safe also regarding S3 we are using localstack and appreciate Stefanos contribution that made it possible to easily override the AWS endpoint.
